apps/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ set (APPS_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 
 set (APPS_SHARE_DIR "${CMAKE_CURRENT_BINARY_DIR}/share")
 
+# If not provided, add default value for resource table
+if(NOT DEFINED ${RSC_TABLE})
+  set(RSC_TABLE 0x3ed20000 CACHE STRING "")
+endif(NOT DEFINED ${RSC_TABLE})
+
```

Comment:
This part is related to the machine, the resource table is optional

Reply:
ok thanks for the review. is it if we keep this but move it to the machine-specific CMakeLists.txt files?

Reply:
updated so that now this logic is only in the machine